### PR TITLE
Blog about all the things LWMIQ

### PR DESCRIPTION
**Pull request by syncrou**

### CODE REVIEW

**Comment by syncrou** [comment]
Fixing the bad urls - sorry about that.

**Comment by JPrause** [comment]
LTGM :+1:

**Comment by JPrause** [comment]
@d-m-u @imtayadeway can you also review.

**Comment by d-m-u** [comment]
Love it. Great style. Like the easter egg, too. 

| a users' tags
I think the apostrophe goes before the 's'? Maybe?

File: site/_posts/2017-01-10-blog-about-all-the-things.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+---
+title: "Last Week in ManageIQ: Blog about all the things"
+author: syncrou
+date: 2017-01-10 09:20:19 -05:00
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hi! [syncrou][me] here to continue our discussions as to what happened in the ManageIQ world last week. As the hours passed by and minutes zipped on
```
Comment (jrafanie) [inline site/_posts/2017-01-10-blog-about-all-the-things.md]: comma after zipped on?  Maybe?

Comment (syncrou) [inline site/_posts/2017-01-10-blog-about-all-the-things.md]: Yeah - That will make it read better. I'll get it changed.

File: site/_posts/2017-01-10-blog-about-all-the-things.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+published: true
+tags: LWIMIQ
+---
+
+Hi! [syncrou][me] here to continue our discussions as to what happened in the ManageIQ world last week. As the hours passed by and minutes zipped on,
+we all thought that a week couldn't possibly justify a blog post. Au contraire, this post is about our ability to [manage all the things][manageiq]!
+
+In the last week we not only continued to manage more things, we did it better! While some thought that there was room for
+[improvement][issues], others (everyone) agreed on one overarching theme. As long as infrastructure and clouds and subnets and networks and hybrid clouds
+and computers and phones and IFTT and general things based on technology exist ([breathe][breath]),
```
Comment (jrafanie) [inline site/_posts/2017-01-10-blog-about-all-the-things.md]: >general things based on technology exist

Seems wordy... maybe:
"technology" things exist

File: site/_posts/2017-01-10-blog-about-all-the-things.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+tags: LWIMIQ
+---
+
+Hi! [syncrou][me] here to continue our discussions as to what happened in the ManageIQ world last week. As the hours passed by and minutes zipped on,
+we all thought that a week couldn't possibly justify a blog post. Au contraire, this post is about our ability to [manage all the things][manageiq]!
+
+In the last week we not only continued to manage more things, we did it better! While some thought that there was room for
+[improvement][issues], others (everyone) agreed on one overarching theme. As long as infrastructure and clouds and subnets and networks and hybrid clouds
+and computers and phones and IFTT and general things based on technology exist ([breathe][breath]),  
+together we shall unite them all under one [manageiq][manageiq]!
```
Comment (jrafanie) [inline site/_posts/2017-01-10-blog-about-all-the-things.md]: all under one manageiq <roof>   

Maybe?

File: site/_posts/2017-01-10-blog-about-all-the-things.md (hunk 2 above)
Comment (syncrou) [inline site/_posts/2017-01-10-blog-about-all-the-things.md]: @jrafanie - If I'm not being overly wordy - then something is clearly and totally wrong! - I'll get it changed.

File: site/_posts/2017-01-10-blog-about-all-the-things.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+[ManageIQ][PRs merged last week] 42 PRs merged.
+[Classic UI][classic ui PRs merged last week] 35 PRs merged.
+[SUI][sui PRs merged last week] 8 PRs merged.
+[Amazon][amazon PRs merged last week] 4 PRs merged.
+[Content][content PRs merged last week] 2 PRs merged.
+[ManageIQ][Commits merged last week] 100 commits pushed (base repo.)
+
+If you're ever looking to help jump into our great project, we just lowered the barrier to entry.
+All [issues][issues], PRs, specs are now broken up by repo.  If you were ever looking for a great reason to manage your managable item, NOW is
```
Comment (d-m-u) [inline site/_posts/2017-01-10-blog-about-all-the-things.md]: Managable needs an 'e'.

File: site/_posts/2017-01-10-blog-about-all-the-things.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+this repo has been separate for some time, the settings for this provider can now be handled by the provider.
+
+## Deleted a major thing
+
+Time.  We deleted time from the development process.
+Each repo holds its own tests, which means both development and merging take less time.
+
+There isn't a way to quantify the time saved, however...
+
+A happy developer continues to develop.  For our project that is a deletion worth
```
Comment (d-m-u) [inline site/_posts/2017-01-10-blog-about-all-the-things.md]: Maybe "For our project, that is a deletion worth fighting for."

File: site/_posts/2017-01-10-blog-about-all-the-things.md (hunk 2 above)
Comment (jrafanie) [inline site/_posts/2017-01-10-blog-about-all-the-things.md]: 😉

**Comment by JPrause** [comment]
@syncrou if all reviews and their edits are complete,..let @hayesr know we're ready for merge. I see the tests have completed as well.

**Pull request merged**